java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -182,7 +184,12 @@ class ReferenceModeStore private constructor(
         callback: ProxyCallback<RefModeStoreData, RefModeStoreOp, RefModeStoreOutput>
     ): Int = callbacks.register(callback)
 
-    override fun off(callbackToken: Int) = callbacks.unregister(callbackToken)
+    override fun off(callbackToken: Int) {
+        if (containerCallbackToken != -1) {
+            containerStore.off(callbackToken)
+        }
```

Comment:
should this be `containerStore.off(containerCallbackToken)`? or really.. if the callbacks manager is empty, that's when we should call `containerStore.off(containerCallbackToken)`, right?

Reply:
+1 callbackToken here seems typo.

Reply:
yup, good catch, typo :)